common/src/main/resources/application.common.properties

```diff
@@ -1,6 +1,6 @@
 efs.mount=${AB2D_EFS_MOUNT:#{systemProperties['java.io.tmpdir']+'/jobdownloads'}}/
 
-audit.files.ttl.hours=24
+audit.files.ttl.hours=72
```

Comment:
Kind of wonder why this is in common and not the audit module.

Reply:
It's in both worker and audit. Worker uses it to set the timeout value in the database when it completes the job.